friendshipper/src/lib/components/playtests/PlaytestCard.svelte

```diff
@@ -285,6 +285,13 @@
 						>{playtest.spec.playersPerGroup}</span
 					>
 				</span>
+				{#if playtest.metadata.creationTimestamp}
+					<span class="text-center font-bold text-sm"
+						>created: <span class="text-primary-400 font-normal"
+							>{new Date(playtest.metadata.creationTimestamp).toLocaleString()}</span
+						>
+					</span>
+				{/if}
```

Comment:
small enhancement to include playtest creation date in UI, should have opened this separately but forgot oops!